mastodon_bot_combined/src/bot2.js

```diff
@@ -0,0 +1,63 @@
+const M = require("./mastodon");
+
+function toot(content, id) {
+  const params = {
+    status: content
+  }
+  if (id) {
+    params.in_reply_to_id = id;
+  }
+  M.post('statuses', params, (error, data) => {
+    if (error) {
+      console.error(error);
+    } else {
+      //fs.writeFileSync('data.json', JSON.stringify(data, null, 2));
+      //console.log(data);
+      console.log(`ID:${data.id}and timestamp:${data.created_at} `);
+      console.log(data.content);
+    }
+  });
+}
+
+module.exports = function bot2() {
+  const listener = M.stream("streaming/user");
+  listener.on("error", err => console.log(err));
+
+  listener.on("message", msg => {
```

Comment:
We probably need to reconcile this with the treegen replies. There are four things that can happen:

1. favorite
2. reblog
3. question
4. generate a tree

If none of these are in a mention, maybe the bot says something more than missing angle?